doc/CONTRIBUTE.md

```diff
@@ -59,9 +59,46 @@ Source folders and files should be named in kebab-case (e.g. `src/trigger-signat
 
 ## System Description
 
-### System Components Diagram
-
-![DEVEN Telemetry system components diagram](./assets/deven-org_telemetry-functions_diagram.jpg)
+### System Flow Diagram
+
+```mermaid
+flowchart TB
+        direction TB
+        Legend --> architecture
+        subgraph Legend
+        Start([Name]) --> |Start/End|empty1[ ]
+        StartPlanned([Name]) --> |Not implemented Start/End|empty1[ ]
+        Process[Name] -->|Process|empty1[ ]
+        Decision{Value/Question} -->|Decision|empty1[ ]
+        Data[/Name/] -->|Data|empty1[ ]
+        Database[(Name)] -->|Database |empty1[ ]
+        style empty1 display:none;
+        start1[ ] --->|Connector| stop1[ ]
+        style start1 display:none;
+        style stop1 display:none;
+        style StartPlanned stroke-dasharray: 5 5,stroke:#aaa;
+        end
+        subgraph architecture [Architecture Overview]
+            direction LR
+            GithubEventStart([Github Event]) -->EventObject[/Event Object/]
+            ManualRunEventStart([Manual Run]) --> EventObject
+            CronJobEventStart([Cron Job]) --> EventObject
+            EventObject --> CheckRequestType[Check request type]
+            CheckRequestType -->|parse| ToBeDefined[To be defined] 
+            CheckRequestType -->|collect| AddSignature[Add Signature]
+            AddSignature --> SignedEventObject[/Signed Event Object/]
+            SignedEventObject --> CheckMetricCondition[Check Metric Condition]
+            CheckMetricCondition --> IsValidCondition{Is valid condition?}
```

Comment:
Nitpick: I'm missing the one-to-many connection here, showcasing that there can be more than one metric collected

Reply:
As this is a flow diagram and not an entity relationship diagram, we don't have a 'one to many' connection.
But this is still a valid point and I will adapt the flow to represent this.

Reply:
The flow is updated. I now added a loop to the flow to show that we do specific thins multiple times.
Also found a better way to integrate not shown links without hiding them over CSS (`~~~`)